packages/clay-form/stories/Input.stories.tsx

```diff
@@ -12,7 +12,7 @@ export default {
 	argTypes: {
 		sizing: {
 			control: {type: 'select'},
-			options: ['lg', 'sm'],
+			options: ['lg', 'regular', 'sm'],
```

Comment:
Maybe this could be an option in the API directly instead of just in the storybook? we can add `regular` and if it is passed, we leave it as the default value. What do you think?

Reply:
I don't have a strong opinion on this. Do you think it will make it clearer for everyone to understand sizing if we add a regular value? I always thought the default was implied if the sizing attribute wasn't specified.

Reply:
Well, I thought that too when `sizing` is not defined it would imply that it is using the regular value but I think that when you are defining the size it is strange for you to look at it and see that there is only lg and sm this is something else implicit than explicit so maybe we can just make it more explicit I think this is more aesthetic too.

Reply:
I think adding regular may make sense, but if we want to add that we should probably be applying that consistently. For instance button currently has the options: `null | xs | sm`. @drakonux were you thinking the `regular` option should be applied everywhere?

Reply:
Oh yes I agree with that, it makes a lot of sense to do this for all components.

Reply:
@matuzalemsteles should we make these updates on this PR?

Reply:
@drakonux do you think it makes sense to make this change across all components?

Reply:
Yes @ethib137 . Let's make a decision and apply it consistently. We are using regular buttons and also for inputs in the Figma library. 

Our top priority is maintaining a consistent approach with Clay and ensuring a one-to-one correspondence at all components.

Reply:
Okay buttons and inputs sound good. Are there any other usages of regular that we should apply this too?

Reply:
I went ahead and added it to the Input and Button. We have some other components with `size` as well:

- Table
- Sheet
- LoadingIndicator
- Modal
- MultiSelect
- Pagination
- PaginationBar
- Panel
- Sticker

Does it make sense to add to these too? I would say yes since we are talk for consistency here but I will wait for other opinions to see if I'm missing something.

Reply:
@drakonux what do you think?

Reply:
Okay, I think we can move forward with this and if it looks good we can do this for the other components just to not block this ticket any longer.